<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>PasteSettings</class>
	<widget class="QDialog" name="PasteSettings">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>300</width>
				<height>140</height>
			</rect>
		</property>
		<property name="sizePolicy">
			<sizepolicy hsizetype="Fixed" vsizetype="Fixed"/>
		</property>
		<property name="minimumSize">
			<size>
				<width>300</width>
				<height>140</height>
			</size>
		</property>
		<property name="maximumSize">
			<size>
				<width>300</width>
				<height>140</height>
			</size>
		</property>
		<property name="windowTitle">
			<string>Paste Settings</string>
		</property>
		<layout class="QVBoxLayout" name="verticalLayout">
			<item>
				<widget class="QGroupBox" name="group_url">
					<property name="sizePolicy">
						<sizepolicy hsizetype="Fixed" vsizetype="Fixed"/>
					</property>
					<property name="minimumSize">
						<size>
							<width>280</width>
							<height>0</height>
						</size>
					</property>
					<property name="title">
						<string>Upload</string>
					</property>
					<layout class="QVBoxLayout" name="verticalLayout">
						<item>
							<layout class="QFormLayout" name="formLayout">
								<item row="1" column="0">
									<widget class="QLabel" name="label_url">
										<property name="minimumSize">
											<size>
												<width>40</width>
												<height>0</height>
											</size>
										</property>
										<property name="text">
											<string>URL:</string>
										</property>
									</widget>
								</item>
								<item row="1" column="1">
									<widget class="QLineEdit" name="input_url"/>
								</item>
							</layout>
						</item>
						<item>
							<layout class="QVBoxLayout" name="verticalLayout">
								<item>
									<widget class="QCheckBox" name="check_copylink">
										<property name="text">
											<string>Copy image link to clipboard</string>
										</property>
									</widget>
								</item>
							</layout>
						</item>
					</layout>
				</widget>
			</item>
			<item>
				<widget class="QDialogButtonBox" name="buttonBox">
					<property name="orientation">
						<enum>Qt::Horizontal</enum>
					</property>
					<property name="standardButtons">
						<set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<tabstops>
		<tabstop>input_url</tabstop>
		<tabstop>check_copylink</tabstop>
	</tabstops>
	<resources/>
	<connections>
		<connection>
			<sender>buttonBox</sender>
			<signal>accepted()</signal>
			<receiver>PasteSettings</receiver>
			<slot>accept()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>236</x>
					<y>299</y>
				</hint>
				<hint type="destinationlabel">
					<x>157</x>
					<y>274</y>
				</hint>
			</hints>
		</connection>
		<connection>
			<sender>buttonBox</sender>
			<signal>rejected()</signal>
			<receiver>PasteSettings</receiver>
			<slot>reject()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>286</x>
					<y>299</y>
				</hint>
				<hint type="destinationlabel">
					<x>286</x>
					<y>274</y>
				</hint>
			</hints>
		</connection>
	</connections>
</ui>
